monetio/sat/_tempo_l2_no2_mm.py

```diff
@@ -0,0 +1,146 @@
+"""TEMPO L2 NO2 data file reader.
+
+History:
+
+
+"""
+
+import logging
+import os
+import sys
+from collections import OrderedDict
+from glob import glob
+from pathlib import Path
+
+import numpy as np
+import xarray as xr
+from netCDF4 import Dataset
+
+
+def _open_one_dataset(fname, variable_dict):
+    """Read locally stored INTELSAT TEMPO NO2 level 2
+    Parameters
+    ----------
+    fname: string
+        fname is local path to netcdf4 (HDF5) file
+
+    variable_dict: dict
+    Returns
+    -------
+    ds: xarray dataset
```

Comment:
```suggestion
    Parameters
    ----------
    fname : str
        Local path to netCDF4 (HDF5) file.
    variable_dict : dict

    Returns
    -------
    ds : xr.Dataset
```
more info: https://numpydoc.readthedocs.io/en/latest/format.html